src/utils/parseError.js

```diff
@@ -32,9 +32,21 @@ export const parseError = (errorObj, {
     ({ errors: errorArray = [error] } = data)
 
     if (shouldLog) {
-      // Log each error provided
-      errorArray.forEach((message) => {
-        console.log(`${name} (${statusCode}): ${message}`)
+      // Log each error provided. Account for when error is a string and
+      // for when error is an object with both path and [errors]
+      errorArray = errorArray.map((message) => {
```

Comment:
An important detail about this change was the change from `forEach` to `map` to ensure the errors nested in an object are readable to clients. Make sure to explain that in comments